src/mobile/dropdown/test/className.test.ts

```diff
@@ -0,0 +1,46 @@
+import { expect, fixture } from "@open-wc/testing";
+import { MobileDropdown } from "../index";
+
+describe("MobileDropdown", () => {
+  describe("className", () => {
+    it("className default prop is null", async () => {
+      const container = new MobileDropdown();
+      const el = await fixture(container);
+      expect(el.classList.length).to.be.equal(0);
+    });
+
+    it("className default prop set successfully", async () => {
```

Comment:
@nmanhit 
Please remove `className` and change the `title` to the same content with expectation.
Ex: `it("should be equal \"options-class\" string when initializing className with \"options-class\" value ")`